### PR TITLE
Version 0.5

### PR DESCRIPTION
**Pull request by ZihengJiang**

### CODE REVIEW

**Comment by yzhliu** [comment]
Shall we also add bug-fixes into the list?

**Comment by ZihengJiang** [comment]
@yzhliu we do not include bug-fix in the news

File: NEWS.md [hunk 1]
```diff
@@ -9,6 +9,76 @@ Refer to the Roadmap issue for complete list on on-going version features.
...
+  - Bundled interpreter demonstration #2297 
+  - Enable PlanMemory in the graph runtime #2120
+- Language Binding
+  - Rust frontend #2292 
+- VTA
+  - Improved RPC for VTA #2043
+- Hybrid python programming model
+  - Support for scheduling #2416 
+  - Support for Inter-function call  #2287 
+- TOP
```
Comment (icemelon9) [inline NEWS.md]: should this be "TOPI"?

File: NEWS.md [hunk 2]
```diff
@@ -9,6 +9,76 @@ Refer to the Roadmap issue for complete list on on-going version features.
...
+  - Intel cpu support
+  - ARM cpu support
+- NVidia GPU 8-bit kernel
+  - int8 gemm recipe
+  - int8 conv2d
+  - Autotvm integration
+- Automated tuning and scheduling
+  - AutoTVM optimizations for mobile GPUs
+  - AutoTVM optimizations for CUDA
+  - AutoTVM for x86
```
Comment (icemelon9) [inline NEWS.md]: Change to "AutoTVM optimizations for x86" as well

File: NEWS.md [hunk 3]
```diff
@@ -9,6 +9,76 @@ Refer to the Roadmap issue for complete list on on-going version features.
...
+  - Rust frontend #2292 
+- VTA
+  - Improved RPC for VTA #2043
+- Hybrid python programming model
+  - Support for scheduling #2416 
+  - Support for Inter-function call  #2287 
+- TOP
+  - Initial support for sparse tensor computation
+  - Improve ARM CPU depthwise convolution performance #2345 
+  - Port winograd ops to relay #2356
```
Comment (Laurawly) [inline NEWS.md]: Shall we include Faster R CNN operator `proposal`?

**Comment by ZihengJiang** [comment]
@Laurawly @icemelon9 @yzhliu Thanks for review. The PR has been merged.

**Pull request merged**